src/ctr_logging.c

```diff
@@ -114,6 +117,15 @@ void configure_log_drivers(gchar **log_drivers, int64_t log_size_max_, int64_t l
 		if (k8s_log_fd < 0)
 			pexit("Failed to open log file");
 
+		struct stat statbuf;
+		if (fstat(k8s_log_fd, &statbuf) == 0) {
+			k8s_bytes_written = statbuf.st_size;
+		} else {
+			nwarnf("Could not stat log file %s, assuming 0 size", k8s_log_path);
+			k8s_bytes_written = 0;
+		}
+		k8s_total_bytes_written = k8s_bytes_written;
```

Comment:
why not write directly to the variable?

Reply:
Both variables are used for different purposes so both need to be set -- did you mean something like `k8s_total_bytes_written = k8s_bytes_written = statbuf.st_size` ?